### ISSUE TITLE
Project contains tens of severe level accessibility issues

### ISSUE THREAD

**Issue by marcushellberg** [vaadin/business-app-starter-flow#7]
Accessibility is a legal requirement for most of the potential companies for this starter. It seems that currently, the starter fails on most accessibility checks and best practices. Please use the axe Chrome accessibility inspector and fix at least the `critical` and `serious` level issues.
